browser/ui/webui/settings/brave_settings_localized_strings_provider.cc

```diff
@@ -79,616 +79,644 @@ const char16_t kEnableNftDiscoveryLearnMoreUrl[] =
+      {"braveSyncResetButton", IDS_BRAVE_SYNC_RESET_BUTTON},
+      {"braveSyncResetConfirmation", IDS_BRAVE_SYNC_RESET_CONFIRMATION},
+      {"braveSyncDeleteAccountButtonAndDialogTitle",
+       IDS_BRAVE_DELETE_SYNC_ACCOUNT_BUTTON_AND_DIALOG_TITLE},
+      {"braveSyncPermanentlyDeleteAccountButton",
+       IDS_BRAVE_SYNC_PERMANENTLY_DELETE_ACCOUNT_BUTTON},
+      {"braveSyncDeleteDeviceConfirmation",
+       IDS_BRAVE_SYNC_DELETE_DEVICE_CONFIRMATION},
+      {"braveSyncPermanentlyDeleteAccountInProgress",
+       IDS_BRAVE_SYNC_PERMANENTLY_DELETE_ACCOUNT_IN_PROGRESS},
+      {"braveSyncDeleteAccountDesc1",
+       IDS_BRAVE_SYNC_DELETE_ACCOUNT_DESCRIPTION_PARTIAL_1},
+      {"braveSyncDeleteAccountDesc2",
+       IDS_BRAVE_SYNC_DELETE_ACCOUNT_DESCRIPTION_PARTIAL_2},
+      {"braveSyncDeleteAccountDesc3",
+       IDS_BRAVE_SYNC_DELETE_ACCOUNT_DESCRIPTION_PARTIAL_3},
+      {"braveSyncFinalSecurityWarning",
+       IDS_BRAVE_SYNC_FINAL_SECURITY_WARNING_TEXT},
+      {"braveSyncPassphraseDecryptionErrorUnlockedSsMessage",
+       IDS_BRAVE_SYNC_PASSPHRASE_DECRYPTION_SS_UNLOCKED_ERROR_MESSAGE},
+      {"braveSyncLeaveAndRejoinTheChainButton",
+       IDS_BRAVE_SYNC_LEAVE_AND_REJOIN_THE_CHAIN_BUTTON},
+      {"braveIPFS", IDS_BRAVE_IPFS_SETTINGS_SECTION},
+      {"braveWeb3", IDS_BRAVE_WEB3_SETTINGS_SECTION},
+      {"braveWeb3Domains", IDS_BRAVE_WEB3_DOMAINS_SETTINGS_SECTION},
+      {"braveTor", IDS_BRAVE_TOR_SETTINGS_SECTION},
+      {"braveWallet", IDS_BRAVE_WALLET_SETTINGS_SECTION},
+      {"braveHelpTips", IDS_SETTINGS_HELP_TIPS},
+      {"braveHelpTipsWaybackMachineLabel",
+       IDS_SETTINGS_HELP_TIPS_SHOW_BRAVE_WAYBACK_MACHINE_PROMPT},
+      {"braveHelpTipsWarnBeforeClosingWindow",
+       IDS_SETTINGS_WINDOW_CLOSING_CONFIRM_OPTION_LABEL},
+      {"braveHelpTipsClosingLastTab",
+       IDS_SETTINGS_CLOSING_LAST_TAB_OPTION_LABEL},
+      {"braveDisableClickableMuteIndicators",
+       IDS_SETTINGS_DISABLE_CLICKABLE_MUTE_INDICATORS},
+
+      // Leo Assistant Page
+      {"leoAssistant", IDS_SETTINGS_LEO_ASSISTANT},
+      {"braveLeoAssistantShowIconOnToolbarLabel",
+       IDS_SETTINGS_LEO_ASSISTANT_SHOW_ICON_ON_TOOLBAR_LABEL},
+      {"braveLeoAssistantResetAndClearDataLabel",
+       IDS_SETTINGS_LEO_ASSISTANT_RESET_AND_CLEAR_DATA_LABEL},
+      {"braveLeoAssistantResetAndClearDataConfirmationText",
+       IDS_SETTINGS_LEO_ASSISTANT_RESET_AND_CLEAR_DATA_CONFIRMATION_LABEL},
+      {"braveLeoAssistantAutocompleteLink",
+       IDS_SETTINGS_LEO_ASSISTANT_AUTOCOMPLETE_LINK},
+      {"leoClearHistoryData",
+       IDS_SETTINGS_LEO_ASSISTANT_CLEAR_HISTORY_DATA_LABEL},
+      {"leoClearHistoryDataSubLabel",
+       IDS_SETTINGS_LEO_ASSISTANT_CLEAR_HISTORY_DATA_SUBLABEL},
+      {"braveLeoAssistantModelSelectionLabel",
+       IDS_SETTINGS_LEO_ASSISTANT_MODEL_SELECTION_LABEL},
+      {"braveLeoModelCategory-chat", IDS_CHAT_UI_MODEL_CATEGORY_CHAT},
+      {"braveLeoModelSubtitle-chat-basic", IDS_CHAT_UI_CHAT_BASIC_SUBTITLE},
```

Comment:
This is the only line I actually changed in this file for this PR (aside from the formatting) - key changed from "-default" to "-basic" suffix.